src/app/clusters/webrtc-transport-provider-server/webrtc-transport-provider-cluster.cpp

```diff
@@ -65,90 +64,79 @@ namespace app {
 namespace Clusters {
 namespace WebRTCTransportProvider {
 
-WebRTCTransportProviderServer::WebRTCTransportProviderServer(Delegate & delegate, EndpointId endpointId) :
-    AttributeAccessInterface(MakeOptional(endpointId), WebRTCTransportProvider::Id),
-    CommandHandlerInterface(MakeOptional(endpointId), WebRTCTransportProvider::Id), mDelegate(delegate)
+WebRTCTransportProviderServer::WebRTCTransportProviderServer(EndpointId endpointId, Delegate & delegate) :
+    DefaultServerCluster({ endpointId, WebRTCTransportProvider::Id }), mDelegate(delegate)
 {}
 
-WebRTCTransportProviderServer::~WebRTCTransportProviderServer()
+DataModel::ActionReturnStatus WebRTCTransportProviderServer::ReadAttribute(const DataModel::ReadAttributeRequest & request,
+                                                                           AttributeValueEncoder & encoder)
 {
-    Shutdown();
-}
-
-CHIP_ERROR WebRTCTransportProviderServer::Init()
-{
-    ReturnErrorOnFailure(CommandHandlerInterfaceRegistry::Instance().RegisterCommandHandler(this));
-    VerifyOrReturnError(AttributeAccessInterfaceRegistry::Instance().Register(this), CHIP_ERROR_INCORRECT_STATE);
-
-    return CHIP_NO_ERROR;
-}
-
-void WebRTCTransportProviderServer::Shutdown()
-{
-    CommandHandlerInterfaceRegistry::Instance().UnregisterCommandHandler(this);
-    AttributeAccessInterfaceRegistry::Instance().Unregister(this);
-}
-
-// AttributeAccessInterface
-CHIP_ERROR WebRTCTransportProviderServer::Read(const ConcreteReadAttributePath & aPath, AttributeValueEncoder & aEncoder)
-{
-    // The only attribute from the spec is "CurrentSessions" (attribute ID 0x0000),
-    // which is a list[WebRTCSessionStruct].
-    if (aPath.mClusterId == Id && aPath.mAttributeId == Attributes::CurrentSessions::Id)
+    switch (request.path.mAttributeId)
     {
-        // We encode mCurrentSessions as a list of WebRTCSessionStruct
-        return aEncoder.EncodeList([this](const auto & encoder) -> CHIP_ERROR {
+    case Attributes::CurrentSessions::Id:
```

Comment:
you already use `chip::app::Clusters::WebRTCTransportProvider::Attributes;` and `chip::app::Clusters::WebRTCTransportProvider` namespaces.

Several namespace items can be reduced throughout this file.